### ISSUE TITLE
DwC JQuery Client - Map View: onClick() Markers Event

### ISSUE THREAD

**Issue by coreyo** [datadavev/DwC_views#38]
Create a hook that allows for the specification of a callback function when the user clicks on a marker.  In the case of a full DwCViews suite, a default action should be created which opens the single record view and displays all of the information for the marker's corresponding record.

**Comment by datadavev** [comment]
Wondering if it is feasible to scroll the records table view to the page of the record that is in the current record view.

So, in map view click on marker -> view record. Switch to records view, and page containing the single record is shown.

Not sure how to calculate what page to record would be on.

**Comment by coreyo** [comment]
Ewww!  While that might be desired for some reason, I would imagine that would get really annoying really quickly for most use.  If you really like the idea, we could extend the recordsTable API to make this easy enough to implement, then the user could add it as an onMarkerClick() event when they initialize the plugin.  However, you are the boss.

**Comment by datadavev** [comment]
Yeah, after thinking it through, that wouldn't really be any use to the user. 

The reverse would be though - highlight the currently selected single record on the map view. Could be done by changing the marker properties, perhaps color to highlight it.
